### PR TITLE
LLM: store the nightly test and pr results separately

### PR DESCRIPTION
**Pull request by WeiguangHan**
@hkvision Store the csv results of arc perf test generated by nightly test and pr separately.

### CODE REVIEW

**Comment by hkvision** [comment]
Also need to modify here: https://github.com/intel-analytics/BigDL/blob/main/.github/workflows/llm_performance_tests.yml#L15

File: .github/workflows/llm_performance_tests.yml [hunk 1]
```diff
@@ -132,11 +134,13 @@ jobs:
...
-          cp ./*.csv /mnt/disk1/nightly_perf_gpu/
+          cp ./*.csv $CSV_SAVE_PATH
           cd ../../../test/benchmark
-          python csv_to_html.py -f /mnt/disk1/nightly_perf_gpu/
+          python csv_to_html.py -f $CSV_SAVE_PATH
           cd ../../dev/benchmark/all-in-one/
-          curl -T ./*.csv ${LLM_FTP_URL}/llm/nightly_perf/gpu/
+          if [ ${{ github.event.schedule}} ]; then
+            curl -T ./*.csv ${LLM_FTP_URL}/llm/nightly_perf/gpu/
+          fi
```
Comment (hkvision) [inline .github/workflows/llm_performance_tests.yml:144]: remove one empty line

Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml:144]: ok

File: .github/workflows/llm_performance_tests.yml [hunk 2]
```diff
@@ -13,11 +13,9 @@ on:
     branches: [main]
     paths:
       - ".github/workflows/llm_performance_tests.yml"
-      - ".github/workflows/llm-binary-build.yml"
-      - ".github/actions/llm/setup-llm-env/action.yml"
-      - ".github/actions/llm/remove-llm-env/action.yml"
-      - ".github/actions/llm/download-llm-binary/action.yml"
-      - "python/llm/test/benchmark/**"
+      - "python/llm/test/benchmark/arc-perf-test.yaml"
```
Comment (hkvision) [inline .github/workflows/llm_performance_tests.yml:16]: should include other yamls as well since this workflow also contains spr perf?

Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml:16]: ok

**Review by hkvision — APPROVED** [review]

**Pull request merged**